### DISCUSSION TITLE
Nachtansicht / Night view

### DISCUSSION THREAD

**Discussion by ToToGSARider** [devemux86/cruiser#629] [Q&A] [answered]
Gibt es eigentlich eine Einstellung, dass die App in einen Nachtmodus wechselt, dass das Display nicht so blendet zum Beispiel im Tunnel?
Habe da nichts gefunden. 

Is there actually a setting for the app to switch to night mode so that the display doesn't dazzle in the tunnel, for example?
I haven't found anything.

**Comment by devemux86** [comment, marked as answer]
@ToToGSARider 

There is currently no automatic function, but it is on my list for implementation:

Derzeit gibt es keine automatische Funktion, aber sie steht auf meiner Liste für die Implementierung:

- https://github.com/devemux86/cruiser/discussions/126

**Comment by Maverikk65** [comment]
Hello, yes I was wondering yesterday about the night mode as well... As I am underway by car it is not a real problem for me to switch the view manually, I don't care about tunnels too much... Also my companion/codriver could do this... 

But I have to figure out the best/easiest way how to switch... Any hints and help is welcome here ;-) 
Danke, VG, NG.

**Reply by devemux86** [reply]
@Maverikk65 A nice solution from @MotoUKRider using the app profiles is described here:
- https://github.com/devemux86/cruiser/discussions/126#discussioncomment-9568315

**Reply by WolfgangZ42** [reply]
> But I have to figure out the best/easiest way how to switch

> A nice solution from @MotoUKRider using the app profiles

Okay - but still 4 clicks (and to maintain an additional profile). With 4 clicks you may do it the normal way either:

<img width="120" src="https://github.com/user-attachments/assets/09ac33c8-5692-444c-9040-5d6bd1a495fe"/>. .<img width="120" src="https://github.com/user-attachments/assets/19df4485-a873-430b-ada0-92d2465dccfd"/>. .<img width="120" src="https://github.com/user-attachments/assets/138199cd-e52c-479c-b8fe-f7010bde79e4"/>. .<img width="120" src="https://github.com/user-attachments/assets/b6204a86-a127-4612-be5c-3fbd2c415fbc"/>. .<img width="120" src="https://github.com/user-attachments/assets/1c920dd8-cdff-4022-b9c1-d4d047615463"/>

Btw: I never use night vision anyway.

**Reply by MotoUKRider** [reply]
The way Cruiser uses Profiles has kind of led to this, so there may need to be a think about how a day/night scenario is handled within the App (via the light sensor or explicit user option). Perhaps as a "child" parameter of the Profile, you have a `Colour Scheme` of Day+Night?

I've updated the screenshot in the Wiki, and replicated it below - My main reason for a good (and safe) night profile is that I sometimes use it in the car if I'm confident that I won't need traffic updates.

![cruiser-navigation-day-and-night-s](https://github.com/user-attachments/assets/4e766409-bf37-4a29-b87c-a26b3c89f7bb)

**Reply by kds112** [reply]
Maybe you can add a menu tile (switch dark/light mode) when long-press the follow button? In the moment there is just the start follow mode tile and there is some space left...

**Reply by Ciutnik** [reply]
@kds112 
This would also require a settings menu to select themes for light and dark. Since it is possible to use your own themes, not everyone uses the default ones 🙂

**Reply by kds112** [reply]
You are right, but then you toggle between the two marked profiles quite easily while driving... Better solution would be the light sensor anyway.

**Comment by devemux86** [comment]
Please let's continue the discussion in the old thread, where more details are mentioned:
- https://github.com/devemux86/cruiser/discussions/126

@WolfgangZ42 @MotoUKRider Maybe you can post your screenshots there again.
